dll/cpl/access/display.c

```diff
@@ -196,7 +229,14 @@ DisplayPageProc(HWND hwndDlg,
                     i = SendDlgItemMessage(hwndDlg, IDC_CURSOR_BLINK_TRACK, TBM_GETPOS, 0, 0);
                     pGlobalData->uCaretBlinkTime = (12 - (UINT)i) * 100;
                     KillTimer(hwndDlg, ID_BLINK_TIMER);
-                    SetTimer(hwndDlg, ID_BLINK_TIMER, pGlobalData->uCaretBlinkTime, NULL);
+                    if (200 <= pGlobalData->uCaretBlinkTime && pGlobalData->uCaretBlinkTime < 1200)
```

Comment:
(Comment for this CPL and the other one):
If, everytime you change the `pGlobalData->uCaretBlinkTime`, you do the automatic sanitization, this may allow simplifying all those range checks afterwards.

Reply:
@HBelusca Thanks for your review